src/stacked_borrows.rs

```diff
@@ -1,10 +1,10 @@
-use std::cell::RefCell;
```

Comment:
Could you please make this mod public for priroda?

Reply:
I need to get access to `Borrow` and `Stacks`, because:

```
[...]
error[E0308]: mismatched types                                                                                                                                                               
   --> src/render/locals.rs:325:64                                                                                                                                                           
    |                                                                                                                                                                                        
325 |     match (ecx.memory().get(ptr.alloc_id), ecx.memory().get_fn(ptr)) {                                                                                                                 
    |                                                                ^^^ expected enum `miri::stacked_borrows::Borrow`, found ()                                                             
    |                                                                                                                                                                                        
    = note: expected type `miri::Pointer<miri::stacked_borrows::Borrow>`                                                                                                                     
               found type `miri::Pointer<()>`
[...]
```

And I can't use `<Evaluator<'tcx> as Machine>::PointerTag`, because it changes lifetime inference in such a way that I get an error.

Reply:
Also it would be nice to be able to visualize what stacked borrows is doing.

Reply:
I added a commit that should help.